features/console.feature

```diff
@@ -1,22 +1,30 @@
 Feature: The console command
   Most aspects of connection to a server are tested in shell feature.
 
-  Scenario: Opening a local Rails console with an irb version between 1.2 and 1.2.6
-    Given the irb version is "1.2"
+  Scenario: Opening a local Rails console with an irb version < 1.2.0
+    Given the irb version is "1.1.0"
     When I run `geordi console`
     Then the output should contain "# Opening a local Rails console"
-      And the output should contain "Util.run! bundle exec rails console -e development -- --nomultiline"
+    And the output should contain "Util.run! bundle exec rails console -e development"
+    But the output should not contain "-- --nomultiline"
+
 
+  Scenario: Opening a local Rails console with an irb version between 1.2.0 and 1.2.6
+    Given the irb version is "1.2.0"
+    When I run `geordi console`
+    Then the output should contain "# Opening a local Rails console"
+      And the output should contain "Util.run! bundle exec rails console -e development -- --nomultiline"
 
   Scenario: Opening a local Rails console with irb version >= 1.2.6
     Given the irb version is "1.2.6"
     When I run `geordi console`
     Then the output should contain "# Opening a local Rails console"
     And the output should contain "Util.run! bundle exec rails console -e development"
+    But the output should not contain "-- --nomultiline"
```

Comment:
This is what I explained in the other comment: with negative expectations, do not be precise but as general as possible. Otherwise, they might easily become falsely satisfied.

```suggestion
    But the output should not contain "nomultiline"
```